package.json

```diff
@@ -26,7 +26,7 @@
     "filecheck": "cd filecheck && ts-node cli.ts",
     "m2h": "ts-node markdown/m2h/cli.ts",
     "md": "echo 'The HTML to Markdown conversion command has been moved to https://github.com/mdn/markdown!' && exit 1",
-    "prepack": "yarn build:dist",
+    "prepack": "yarn build:ssr && yarn build:dist",
```

Comment:
(Could you extract this change to a separate PR, as this goes beyond "update NPM packaging ignores"?)

In the `npm-publish` workflow, we run `build:prepare`, which includes `build:ssr`:

https://github.com/mdn/yari/blob/9b0e626bd075604bcc132f0171a071fc9055cbbb/.github/workflows/npm-publish.yml#L51-L56

If `build:ssr` is really enough, then we could (and have to) merge those steps, as we would otherwise be missing those environment variables when running `yarn build:ssr` (again) as part of the pack. (And my hope is that those environment variables would actually be available in the context of the invocation by pack.)

Same goes for `npm-publish-simulation`:

https://github.com/mdn/yari/blob/9b0e626bd075604bcc132f0171a071fc9055cbbb/.github/workflows/npm-published-simulation.yml#L40-L60